src/main/resources/hibernate.cfg.xml

```diff
@@ -3,8 +3,8 @@
         <property name="hibernate.dialect">org.hibernate.dialect.MySQLDialect</property>
         <property name="connection.url">jdbc:mysql://localhost:3306/hibernate_relations_hw</property>
         <property name="connection.driver_class">com.mysql.cj.jdbc.Driver</property>
-        <property name="connection.username">****</property>
-        <property name="connection.password">********</property>
+        <property name="connection.username">user_name</property>
+        <property name="connection.password">password</property>
         <property name="show_sql">true</property>
         <property name="hbm2ddl.auto">create-drop</property>
 
```

Comment:
I guess it was meant to remove this empty line, rather than providing explicitly your credentials